stamina-core/src/main/scala/stamina/Persisters.scala

```diff
@@ -10,6 +10,11 @@ import scala.reflect.ClassTag
  *
  */
 case class Persisters(persisters: List[Persister[_, _]]) {
+  private val overlappingTags = persisters.groupBy(_.tag).filter(_._2.length > 1).mapValues(_.map(_.key))
+  private def join(strings: Iterable[String]) = strings.reduce(_ + ", " + _)
+  require(overlappingTags.isEmpty, s"Overlapping persisters: " +
+    join(overlappingTags.map(tuple ⇒ "Tags " + join(tuple._2) + " all persist " + tuple._1.runtimeClass)))
+
```

Comment:
Ah, the white spacing made it hard to read. I didn't even see the `require` because it read as part of the body of `join`.